server/controllers/wordpress.js

```diff
@@ -36,6 +38,7 @@ export const articles = async(ctx, next) => {
   const path = ctx.request.url;
   const {page, q} = ctx.request.query;
   const articleStubsResponse = await getArticleStubs(maxItemsPerPage, {page}, q);
+  const globalAlert = ctx.intervalCache.get('globalAlert');
```

Comment:
Why do we still need to do this if we have it in globals?

Reply:
we don't – it slipped through the net. Thanks.